### ISSUE TITLE
URL shortener

### ISSUE THREAD

**Issue by zuphilip** [CrossRef/member-domains#7]
It looks that currently you are not handling shortened link in the `guess-doi` endpoint, e.g. https://t.co/VPy5CKKSpY from https://www.altmetric.com/details/2068207/twitter .

**Comment by afandian** [comment]
Correct, I've not thought of that yet in the DOI Destinations sevice. It's an interesting idea.

In the context of Crossref Event Data, we are actually following shortened links **in tweets** before it hits the DOI Destinations service, so we would still catch them in our Twitter data.

Can you think of a non-twitter use case? I'm happy to add support if it would be useful.

**Comment by zuphilip** [comment]
I have seen shortened urls in social media in general: Facebook, Blogs sometimes even Emails. My use case at some point is to enrich our institutional repository, which lacks currently a lot of dois for older articles. We are not using shortened dois. Thus, no real use case for this issue. (I am just trying to understand better, what the endpoint can do and where the limits are.)

Maybe, it is worth to describe more how you use this service at CrossRef and what is before and after. I could imagine that you are also somehow trying to confirm the guessed dois. Or is this described somewhere in the Crossref Event Data context?

**Comment by afandian** [comment]
It's great that you're interested in using this and I'd like to help improve it in any ways that are useful. Feel free to contact jwass@crossref.org 

Crossref Event Data is an evolving product, so when it's ready for launch there will be full documentation. It's a little complicated as every data source does things slightly differently! E.g. our facebook source doesn't use this service at all (and queries the Facebook API for the landing page).
